internal/store/hpa_test.go

```diff
@@ -111,3 +169,11 @@ func TestHPAStore(t *testing.T) {
 		}
 	}
 }
+
+func int32ptr(value int32) *int32 {
+	return &value
+}
+
+func resourcePtr(quantity resource.Quantity) *resource.Quantity {
+	return &quantity
+}
```

Comment:
Let me know if these helper functions exist somewhere else, I didn't find any at first glance.

Reply:
I don't think we need these helper functions right now. Not a strong opinion though.